utils/github/__main__.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
```

Comment:
Is it common way to run python3 scripts? I'd expect something like `./github` or `./check-prs`

Reply:
No, I just wanted to write the script at first, but ended up with a module. This line is unnecessary.

Reply:
This remark is not refer to this line. Can we add normal script and use it without knowledge about python interpreter options?

Reply:
Removed this line, add separate script wrapper.